README.md

```diff
@@ -455,9 +648,14 @@ Penalty for transmission line limit violations (if transmission violations are e
 Penalty for ending the interval with less stored energy than the start, or reward for ending with more.
 
 [Gurobi]: https://www.gurobi.com
-[Download Gurobi]: https://www.gurobi.com/downloads/gurobi-optimizer-eula/
 [Gurobi Installation Guide]: https://www.gurobi.com/documentation/quickstart.html
+[Gurobi Cloud License]: https://cloud.gurobi.com/manager/licenses
 [Julia]: https://julialang.org/
-[Download Julia]: https://julialang.org/downloads/
-[Zenodo]: https://zenodo.org/record/3530898
-[Python]: https://www.python.org/
\ No newline at end of file
+[Download Julia]: https://julialang.org/downloads/#current_stable_release
+[Python]: https://www.python.org/
+[Download Python]: https://www.python.org/downloads/release/python-386/
+[Docker]: https://docs.docker.com/get-docker/
+[Zenodo]: https://zenodo.org/record/4450759
```

Comment:
Please do not update the Zenodo link. The `3530898` link will always automatically redirect to the latest version of the dataset, any other link points to a specific version.

Reply:
Good to know! It wasn't clear to me which (if any) Zenodo links wouldn't become out of date.